$NetBSD: patch-ae,v 1.1 2006/10/04 18:48:04 rillig Exp $

According to a private mail from Tollef Fog Heen, the pkg-config
maintainer, the test suite has been wrong, but the code ok. This patch
fixes the tests.

--- check/check-requires-private.orig	2006-08-16 19:05:30.000000000 +0200
+++ check/check-requires-private	2006-10-04 20:42:05.000000000 +0200
@@ -9,14 +9,14 @@ fi
 
 . ${srcdir}/common
 
-# expect cflags from requires-test and public-dep
+# expect cflags from requires-test, private-dep and public-dep
 ARGS="--cflags requires-test"
-RESULT="-I/requires-test/include -I/public-dep/include"
+RESULT="-I/requires-test/include -I/private-dep/include -I/public-dep/include"
 run_test
 
 # still expect those cflags for static linking case
 ARGS="--static --cflags requires-test"
-RESULT="-I/requires-test/include -I/public-dep/include"
+RESULT="-I/requires-test/include -I/private-dep/include -I/public-dep/include"
 run_test
 
 # expect libs for just requires-test and public-dep
